pkg/controller/admissionchecks/multikueue/workload.go

```diff
@@ -316,15 +362,19 @@ func (a *wlReconciler) reconcileGroup(ctx context.Context, group *wlGroup) error
 			}
 			// update the message
 			acs.Message = fmt.Sprintf("The workload got reservation on %q", reservingRemote)
+			// update the transition time
+			acs.LastTransitionTime = metav1.NewTime(time.Now())
+
 			wlPatch := workload.BaseSSAWorkload(group.local)
 			workload.SetAdmissionCheckState(&wlPatch.Status.AdmissionChecks, *acs)
 			err := a.client.Status().Patch(ctx, wlPatch, client.Apply, client.FieldOwner(ControllerName), client.ForceOwnership)
 			if err != nil {
-				return err
+				return reconcile.Result{}, err
 			}
 		}
-		// drop this if we want to create new remote workloads while holding a reservation
-		return nil
+		// drop this if we want to create new remote workloads while holding a reservation.
+		// check again the connection to the remote in workerLostTimeout.
+		return reconcile.Result{RequeueAfter: a.workerLostTimeout}, nil
```

Comment:
As in the other comment, I think we should check the connection more often than in 15min.